pkg/health/clusterStatsHandler.go

```diff
@@ -304,6 +327,63 @@ func getIngestionStats(myid uint64) (map[string]utils.ResultPerIndex, int64, flo
 	return ingestionStats, totalEventCount, totalIncomingBytes, totalOnDiskBytes
 }
 
+func GetTracesStats(myid uint64) (map[string]utils.ResultPerIndex, int64, float64, float64) {
+
+	totalTraceBytes := float64(0)
+	totalTraceSpanCount := int64(0)
+	totalTraceOnDiskBytes := float64(0)
+
+	traceStats := make(map[string]utils.ResultPerIndex)
+	allVirtualTableNames, err := vtable.GetVirtualTableNames(myid)
+	traceIndices := make([]string, 0)
+
+	for k := range allVirtualTableNames {
+		if isTraceRelatedIndex(k) {
+			traceIndices = append(traceIndices, k)
+		}
+	}
+	sort.Strings(traceIndices)
+
+	if err != nil {
+		log.Errorf("GetTracesStats: Error in getting virtual table names, err:%v", err)
+	}
+
+	allvtableCnts := segwriter.GetVTableCountsForAll(myid)
```

Comment:
nit: use `counts` instead of `cnts`. Also `allvtable` should be `allVtable` or maybe `allVTable`